src/tools/run-make-support/src/rustc.rs

```diff
@@ -40,6 +45,14 @@ impl Rustc {
     /// Construct a new `rustc` invocation.
```

Comment:
Suggestion: could you also mention it sets `-L cwd()`? I think it would be helpful to link to ``[`bare_rustc`]`` if the test writer wants a rustc invocation without `-L cwd()`.